test/onnx_import.in.cpp

```diff
@@ -38,6 +38,26 @@ static std::string s_manifest = "${MANIFEST}";
 using Inputs = std::vector<std::vector<float>>;
 using Outputs = std::vector<std::vector<float>>;
 
+namespace
```

Comment:
Please copy this function into a separate PR, so we can merge it into `master`.

Reply:
https://github.com/NervanaSystems/ngraph/pull/2423